packages/frontend/editor-ui/src/components/MainSidebar.vue

```diff
@@ -280,6 +319,20 @@ const handleSelect = (key: string) => {
 		case 'insights':
 			telemetry.track('User clicked insights link from side menu');
 		default:
+			if (key.startsWith('whats-new-article-')) {
+				const articleId = Number(key.replace('whats-new-article-', ''));
```

Comment:
Are `NaN` values expected here? Should we add a bit od error handling?

Reply:
I can't really see any way this would error - we only create these at the mapping function below, and all articles have a required integer ID on them. As long as we don't introduce other elements with the same prefix we're fine.